lib/db.js

```diff
@@ -302,6 +307,22 @@ module.exports = function (
       )
   }
 
+  DB.prototype.sessionTokenVerified = function (id) {
+    log.trace({ op: 'DB.sessionTokenVerified', id: id })
+    return this.pool.get('/sessionToken/' + id.toString('hex') + '/verified')
+      .then(
+        function (result) {
+          return result
```

Comment:
Should we be calling SessionToken.fromHex here to turn the data into a proper SessionToken object, similarly to how the above `sessionToken` method does it?